app/assets/javascripts/geoblacklight/downloaders/downloader.js

```diff
@@ -10,7 +10,7 @@
       L.Util.setOptions(this, options);
       this.$el = $(el);
       this.options.spinner.hide();
-      $('.exports .card-header').append(this.options.spinner);
+      this.$el.before(this.options.spinner);
```

Comment:
the spinner was never appearing for me, I think it is because I couldn't find a DOM element with the `exports` class in it .... this instead places it near the download link